x-pack/plugins/security_solution/common/siem_migrations/model/rule_migration.gen.ts

```diff
@@ -104,6 +105,21 @@ export const ElasticRule = z.object({
 export type ElasticRulePartial = z.infer<typeof ElasticRulePartial>;
 export const ElasticRulePartial = ElasticRule.partial();
 
+/**
+ * The prebuilt rule version.
+ */
+export type PrebuiltRuleVersion = z.infer<typeof PrebuiltRuleVersion>;
+export const PrebuiltRuleVersion = z.object({
+  /**
+   * The latest available version of prebuilt rule.
+   */
+  target: RuleResponse,
```

Comment:
@semd I decided to keep same names as they used within "prebuilt rules". Let me know if you think it will be unclear to other devs.

Reply:
Yup, I like it